lib/checks/label/label-content-name-mismatch.json

```diff
@@ -3,7 +3,7 @@
   "evaluate": "label-content-name-mismatch-evaluate",
   "options": {
     "pixelThreshold": 0.1,
-    "occuranceThreshold": 3
+    "occurrenceThreshold": 3
```

Comment:
Unfortunately this is a breaking change. I'm not sure we can do this until we're ready to ship v5.

Reply:
Agreed. Although what we could do is allow either variation, and deprecate the option with the typo.

Reply:
I like that idea!